tests/aws/services/events/test_events_cross_account_region.py

```diff
@@ -320,3 +318,138 @@ def test_event_bus_to_event_bus_cross_account_region(
         PolicyName=policy_name_bus_primary_to_bus_secondary,
     )
     aws_client.iam.delete_role(RoleName=role_name_bus_primary_to_bus_secondary)
+
+
+class TestEventsCrossAccountRegion:
+    @markers.aws.validated
+    @pytest.mark.skipif(is_old_provider(), reason="Not supported in v1 provider")
+    @pytest.mark.parametrize("cross_scenario", ["account"])
+    @pytest.mark.parametrize("event_bus_type", ["custom"])
+    def test_put_events(
+        self,
+        cross_scenario,
+        event_bus_type,
+        region_name,
+        secondary_region_name,
+        account_id,
+        secondary_account_id,
+        get_primary_secondary_client,
+        snapshot,
+    ):
+        # Create aws clients
+        response = get_primary_secondary_client(cross_scenario)
+        aws_client = response["primary_aws_client"]
+        secondary_aws_client = response["secondary_aws_client"]
+        secondary_region_name = response["secondary_region_name"]
+        secondary_account_id = response["secondary_account_id"]
+
+        # overwriting randomized region https://docs.localstack.cloud/contributing/multi-account-region-testing/
+        # requires manually adding region replacement for snapshot
+        snapshot.add_transformer(snapshot.transform.regex(region_name, "<region>"))
+        snapshot.add_transformer(
+            snapshot.transform.regex(secondary_region_name, "<region-secondary>")
+        )
+        snapshot.add_transformer(snapshot.transform.regex(account_id, "<account>"))
+        snapshot.add_transformer(
+            snapshot.transform.regex(secondary_account_id, "<account-secondary>")
+        )
+
+        # Create event bus in secondary region / account
+        if event_bus_type == "default":
+            event_bus_name = "default"
+            event_bus_arn = f"arn:aws:events:{region_name}:{account_id}:event-bus/default"
+        else:
+            event_bus_name = f"test-bus-{short_uid()}"
+            response = secondary_aws_client.events.create_event_bus(Name=event_bus_name)
+            event_bus_arn = response["EventBusArn"]
+
+        # Allow principal to put event to event bus
+        secondary_aws_client.events.put_permission(
+            StatementId=f"TargetEventBusAccessPermission{short_uid()}",
+            EventBusName=event_bus_name,
+            Action="events:PutEvents",
+            Principal="*",
+        )
+
+        # Create SQS queue in secondary region / account
+        queue_name = f"test-queue-{short_uid()}"
+        queue_url = secondary_aws_client.sqs.create_queue(QueueName=queue_name)["QueueUrl"]
```

Comment:
Alternatively, the `localstack.testing.pytest.fixtures.sqs_create_queue` fixture could be used by configuring it with the secondary_aws_client instead of manually creating and cleaning up the SQS queue.
SO post about indirect parametrization: https://stackoverflow.com/a/33879151

Reply:
more of an idea than suggestion to implement, feel free to skip